internal/awsdata/data.go

```diff
@@ -263,6 +270,19 @@ func AppendIfMissing(slice []string, s string) []string {
 	return append(slice, s)
 }
 
+func HumanReadableBytes(b int64) string {
```

Comment:
Another random util function to be moved to a separate file at some future point.